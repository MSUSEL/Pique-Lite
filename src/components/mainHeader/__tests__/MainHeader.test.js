import React from 'react';
import {shallow} from 'enzyme';

import MainHeader from '../MainHeader.component'

describe('MainHeader Component', () => {
    let wrapper;
    wrapper = shallow(<MainHeader/>);

    it('should render MainHeader component', () => {
        expect(wrapper).toMatchSnapshot();
    })
})