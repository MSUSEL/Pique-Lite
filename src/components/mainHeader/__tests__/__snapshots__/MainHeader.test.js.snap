// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader Component should render MainHeader component 1`] = `ShallowWrapper {}`;

exports[`Sidebar Component should render Sidebar component 1`] = `ShallowWrapper {}`;
